src/utils/store-utils.js

```diff
@@ -130,6 +130,11 @@ function getInstanceConfiguration(instance) {
   if (maxMediaAttachments) {
     statuses.maxMediaAttachments ??= maxMediaAttachments;
   }
+  if(pleroma) {
+    // Set an arbitrary high attachment limit, since neither Akkoma nor Pleroma have an attachment limit.
+    statuses.maxMediaAttachments ??= 1000;
```

Comment:
Honestly this feels hacky 🙈. I would rather have the code check if there's `maxMediaAttachments`, use it, else don't check.

Reply:
It is :see_no_evil: 

Yeah, that's much more reasonable, I'm doing that then!